astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -247,12 +261,25 @@ def main():
 
     opts, args = handle_options(argv)
 
+    if opts.tolerance is not None and opts.rtol is not None:
```

Comment:
In the codes above, `tolerance` overwrites `rtol` with a warning, but here it is outright error. Why the difference in behavior?